### ISSUE TITLE
Cannot configure stanchion to use lager_syslog_backend

### ISSUE THREAD

**Issue by dsabeti** [basho/stanchion#80]
Hi,

We're trying to get our stanchion process to forward logs to a syslog aggregator using lager_syslog_backend. Here is our configuration for stanchion:

```
{handlers, [
                      {lager_file_backend, [{file, "/var/vcap/sys/log/stanchion/error.log"},
                                            {level, error},
                                            {size, 10485760},
                                            {date, "$D0"},
                                            {count, 5}]},
                      {lager_file_backend, [{file, "/var/vcap/sys/log/stanchion/console.log"},
                                            {level, info},
                                            {size, 10485760},
                                            {date, "$D0"},
                                            {count, 5}]},
                      {lager_syslog_backend, ["vcap.stanchion", local1, info]}
                     ]},
```

The error we're getting when we try to start stanchion is:

```
2014-07-02 18:36:13.356 [error] <0.119.0>@lager_handler_watcher:105 
Lager failed to install handler lager_syslog_backend into lager_event, retrying later : 
{'EXIT',
 {undef,
  [{lager_syslog_backend,
    init,
    [["vcap.stanchion",
      local1,
      info]],
    []},
   {gen_event,
    server_add_handler,
    4,
    [{file,
      "gen_event.erl"},
     {line,
      400}]},
   {gen_event,
    handle_msg,
    5,
    [{file,
      "gen_event.erl"},
     {line,
      266}]},
   {proc_lib,
    init_p_do_apply,
    3,
    [{file,
      "proc_lib.erl"},
     {line,
      227}]}]}}
```

So the problem is that lager_syslog_backend cannot be installed.

We tried to dig through some of the riak-cs and stanchion code to see why this might work for riak-cs but not for stanchion. We noticed that `reltool.config` files were different between the two packages: riak-cs contained the following:

```
{sys, [
       {lib_dirs, ["../deps", "../apps"]},
       {rel, "riak-cs", "1.4.4",
        [
         kernel,
         ...
         lager,
         lager_syslog,
         poolboy,
         ...
```

Stanchion, on the other hand, has:

```
{sys, [
       {lib_dirs, ["../deps", "../apps"]},
       {rel, "stanchion", "1.4.3",
        [
         kernel,
         stdlib,
         sasl,
         public_key,
         ssl,
         os_mon,
         crypto,
         runtime_tools,
         mochiweb,
         webmachine,
         lager,
         stanchion
        ]},
```

We're guessing the problem might be that Stanchion is missing `lager_syslog`?

Any advice on setting up stanchion to use syslog would be great.

Thanks,
@dsabeti and @drich10

**Comment by shino** [comment]
@dsabeti Thanks for reporting and digging the issue.

Changes will be similar with syslog support for riak_cs:
https://github.com/basho/riak_cs/pull/617/files

Two changes are:
- adding `lager_syslog` to rebar.config's deps and
- adding `lager_syslog` to reltool.config (as you described).

**Comment by kuenishi** [comment]
Fixed via #81.
